dev-client/src/screens/SiteSettingsScreen/SiteSettingsScreen.tsx

```diff
@@ -106,7 +106,10 @@ export const SiteSettingsScreen = ({siteId}: Props) => {
               icon={<Icon name="info" />}
             />
           </FormControl.Label>
-          <Select dropdownIcon={<Icon name="arrow-drop-down" />} />
+          <ProjectSelect
+            projectId={site.projectId ?? null}
+            setProjectId={_ => {}}
```

Comment:
question: what does `_ => {}` do?

Reply:
oh good catch! that was a placeholder function during an earlier pass, it's a NOOP function but should definitely be an op!

Reply:
ok i went back in, this was already a NOOP before, and implementing it properly now would require a confirmation modal and generally non-trivial changes so i don't want to also pull that into this PR